src/tests/nativeaot/SmokeTests/PInvoke/PInvoke.cs

```diff
@@ -404,6 +404,9 @@ private static void TestArrays()
 
             ThrowIfNotEquals(0, CheckIncremental(arr, ArraySize), "Array marshalling failed");
 
+            int[] empty = new int[0];
```

Comment:
As long as we're here, let's add the `int[] isNull = null;` case as well.

Reply:
Hmmm. Well `char` isn't technically a blittable primitive array, at least in CoreCLR. I don't know about native AOT.

Reply:
We should be running all of the src/tests/Interop pri-0 tests regularly. Don't we have coverage of these edge cases there?

The src/tests/nativeaot/SmokeTests/PInvoke tests are largely historical when we had so little interop support that it wasn't worth to even try running the real interop tests with CoreRT. Since I'm not sure if _everything_ is redundant with src/tests/Interop, we can't delete these, but at some point it might be interesting to move them to src/tests/Interop so that they run on all runtimes (tests under nativeaot are native AOT specific and don't run anywhere else).